### PR TITLE
 add linter files

### PR DESCRIPTION
**Pull request by Alabi12**
add inter files to the project

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+          <div class="list-header">
+            <h3>Your resent score</h3>
+            <button class="refresh-btn">Refresh</button>
+          </div>
+          <ul class="list">
+            <li class="list-item">Name: 50</li>
+            <li class="list-item">Name:40</li>
+            <li class="list-item">Name: 100</li>
+            <li class="list-item">Name: 60</li>
+            <li class="list-item">Name: 90</li>
```
Comment (zieeco) [inline dist/index.html]: - Hey, this part should be populated using javascript as this was not stated in the requirement, please, kindly bear this in mind as I'd mark it optional.

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+.board-content {
+  display: flex;
+  gap: 4rem;
+}
+
+.list-header {
+  display: flex;
+  align-items: center;
+  gap: 6rem;
+}
```
Comment (zieeco) [inline src/style.css]: - Please let's kindly improve our design on this part to match the wireframe as much as we can. see the images below:


*wireframe*
Screenshot from 2022-11-16 00-36-03

*your design*
Screenshot from 2022-11-16 00-34-47

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Alabi12 ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
_Check the comments under the review._

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @Alabi12, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaduKamara — APPROVED** [review]
# Status: APPROVED 🟢🟢🟢 

Hi @Alabi12,

Your project is complete!  

ClappingLeonardoDicaprioGIF (3)

## Highlights:
✅ You used Gitflow correctly
✅ Linter checks passed 
✅ Used async and await and arrow functions when fetching and posting score data to the API 👍 

There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**